### DISCUSSION TITLE
DNS能否指定inbound，即不同的inbound使用不同的DNS配置

### DISCUSSION THREAD

**Discussion by yincu** [v2fly/v2ray-core#1172] [Q&A]
如标题。用于解锁流媒体服务用。不同的inbound可以转发到不同的服务上。这样就可以不启动 多个进程了。

现在的情况是顶层DNS配置是所有inbound共用一套规则。可否根据inbound的tag绑定到一套DNS解析逻辑？

现在应该是不支持吧？后续可能加上这个功能吗？

**Comment by qingfengshi** [comment]
端口转发可以实现你的目的

**Reply by yincu** [reply]
不是我想要的效果啊。而且还是需要内置的DNS配置生效。而不是无脑直接端口转发dns请求到某个dns服务器

**Comment by BozhanL** [comment]
https://github.com/v2fly/domain-list-community/blob/cd7a88df74b676d9bce1149efa85eea8b24e575a/data/netflix#L14-L24
用domains转发到不同地址，看起来可以

**Reply by yincu** [reply]
我需要的不是根据domains转发到不同的地址。我是需要 不同的inbound， 相同的domain可配置转发到不同的DNS上。你这个是我现在用的。

**Comment by Cwek** [comment]
现阶段应该没有，最可行的就是v2的路由根据inbound重定向DNS流量到对应指定地址端口freedom出口上，然后这些freedom对接对应的DNS（或者对应dokodemo的DNS端口转发）上。

**Comment by rurirei** [comment]
you're required the support for the multi DNSObject, which is unsupported.

**Reply by rurirei** [reply]
this is a good feature that i want to implement to.

**Reply by Hosify** [reply]
希望不久以后就能够实现这个功能，实在不想同时运行两个v2ray。

**Comment by whatbugsyou** [comment]
目前没有”不同的inbound走不同的dns配置“”这个功能。因为目前dns只可以配置一个全局的。
